docs/guides/server/directory-structure.adoc

```diff
@@ -0,0 +1,31 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Directory Structure"
+summary="Understand the purpose of the directories under the installation root"
+includedOptions="">
+
+== Installation Locations
+
+`/opt/keycloak` is the root install location for the server in all containerized usage shown for {project_name} including <@links.server id="containers"/> and the Getting Started Guides for Docker, Podman, Kubernetes, and OpenShift - see link:{gettingstarted_link_latest}[{gettingstarted_name_short}].
+
+If you are installing from a zip file then by default there will be an install root directory of `{archivebasename}-{version}`, which can be created anywhere you choose on your filesystem.
```

Comment:
I'd swap this chapter with `Directory Structure`. It feels to me a bit odd to start a "Directory Structure" guide with a note where KC is placed in the container image.

Reply:
I was trying to first establish context about where the directory structure could be found. Would altenative wording help?

Reply:
I'd maybe at least swap the two paragraphs to have ZIP instructions first (at a more "natural" distribution).

Reply:
Ok, made that change.